parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -230,15 +230,6 @@ write_files:
     }
 {{end}}
 
-{{if HasCiliumNetworkPlugin }}
-- path: /etc/systemd/system/sys-fs-bpf.mount
```

Comment:
this file is still used somewhere else or should we remove it too?

Reply:
Removed it now, seems to have only been used if `HasCiliumNetworkPlugin` was true